### ISSUE TITLE
Adjust event store adapters to support 5.0

### ISSUE THREAD

**Issue by codeliner** [prooph/event-store#61]
- [x] [event-store-mongodb-adapter](https://github.com/prooph/event-store-mongodb-adapter)
- [x] [event-store-doctrine-adapter](https://github.com/prooph/event-store-doctrine-adapter)
- [x] ~~[event-store-zf2-adapter](https://github.com/prooph/event-store-zf2-adapter)~~ - Abandoned, see https://github.com/prooph/event-store-zf2-adapter/issues/2

Depends on #60

**Comment by codeliner** [comment]
Decision is made. Adapters will provide a container-aware factory and will be refactored to require all dependencies in the constructor instead of an options array. The adapter helper trait added with #56 will be removed again because it is no longer needed.
